src/stories/pages/Experiment/Experiment.js

```diff
@@ -29,8 +29,20 @@ import { ImageSelectionTask }
  */
 
 export const Experiment = ({
-  title, description, consentForms, url, preSurveys, postSurveys,
+  preSurveys, postSurveys,
```

Comment:
I know this isn't implemented yet, but I would think we would be grabbing the surveys through a firebase request in the useEffect below?